src/main/java/address/util/DragContainer.java

```diff
@@ -0,0 +1,32 @@
+package address.util;
+
+import javafx.scene.input.DataFormat;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ *To contain a list of Integer.
+ */
+public class DragContainer implements Serializable{
```

Comment:
The clipboard needs object to be serialized. This container serves as a convenience class to contain other related info, for e.g. the DataFormat.  In future if we need extension, we could add fields into this class also.